packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -313,7 +313,8 @@ export class LangChainEmbeddingWrapper implements Embedding {
 }
 
 /** A default embedding useful for DocsQA. Note that this requires an OPENAI_API_KEY. */
-export const defaultEmbedding = new LangChainEmbeddingWrapper(new OpenAIEmbeddings());
+const openAIEmbeddings = new OpenAIEmbeddings({ openAIApiKey: process.env.OPENAI_API_KEY });
```

Comment:
Ah, this wasn't getting picked up automatically?

If this env var is required to work, we should throw an exception if it's not found.